include/zephyr/arch/riscv/exp.h

```diff
@@ -49,6 +53,9 @@ struct soc_esf {
 #endif
 #endif
 
+#if defined(CONFIG_RISCV_SOC_ISR_STACKING)
+	SOC_ISR_STACKING_ESF_DECLARE;
+#else
```

Comment:
It seems that better to use #ifdef for single CONFIG_AAA.
But this is just my opinion. Because there is no consistency in original source code...
(ex. CONFIG_RISCV_ISA_RV32E used defined() pattern)

Reply:
I don't think the project enforces that. I personally prefer the `if defined` form because it plays more nicely with the syntax highlighting in my editor.

Reply:
OK, thanks for your comments.